Reduce gif size (#192)
easystats/datawizard: README.Rmd

[![total](https://cranlogs.r-pkg.org/badges/grand-total/datawizard)](https://cranlogs.r-pkg.org/) [![status](https://tinyverse.netlify.com/badge/datawizard)](https://CRAN.R-project.org/package=datawizard)

<img src='https://media.giphy.com/media/VcizxCUIgaKpa/giphy.gif' width="200" align = "right"/>

> ***:sparkles: Hockety pockety wockety wack, prepare this data forth and back***

Author: ah ok I wasn't sure if you wanted to include it :sweat_smile:

> ***:sparkles: Hockety pockety wockety wack, prepare this data forth and back***

Author: but you want to keep the gif, right?

[![total](https://cranlogs.r-pkg.org/badges/grand-total/datawizard)](https://cranlogs.r-pkg.org/) [![status](https://tinyverse.netlify.com/badge/datawizard)](https://CRAN.R-project.org/package=datawizard)

<img src='https://media.giphy.com/media/VcizxCUIgaKpa/giphy.gif' width="200" align = "right"/>

<!-- ***:sparkles: Hockety pockety wockety wack, prepare this data forth and back*** -->

<!-- ***Hockety pockety wockety wock, messy data is in shock*** -->

<!-- ***Hockety pockety wockety woss, you can cite i-it from JOSS*** <sup>(soon)</sup> -->

<!-- ***Hockety pockety wockety wass, datawizard saves your ass! :sparkles:*** -->

> `datawizard` is a lightweight package to easily manipulate, clean, transform, and prepare your data for analysis. It is part of the [easystats ecosystem](https://easystats.github.io/blog/), a suite of R packages to deal with your entire statistical analysis, from cleaning the data to reporting the results.

Reviewer: ```suggestion
`{datawizard}` is a lightweight package to easily manipulate, clean, transform, and prepare your data for analysis. It is part of the [easystats ecosystem](https://easystats.github.io/blog/), a suite of R packages to deal with your entire statistical analysis, from cleaning the data to reporting the results.
```

> `datawizard` is a lightweight package to easily manipulate, clean, transform, and prepare your data for analysis. It is part of the [easystats ecosystem](https://easystats.github.io/blog/), a suite of R packages to deal with your entire statistical analysis, from cleaning the data to reporting the results.

Reviewer: ```suggestion
`datawizard` is a lightweight package to easily manipulate, clean, transform, and prepare your data for analysis. It is part of the [easystats ecosystem](https://easystats.github.io/easystats/), a suite of R packages to deal with your entire statistical analysis, from cleaning the data to reporting the results.
```

Author: done

> Most courses and tutorials about statistical analysis and modelling focus on the different methods and assume that you have a cleaned and tidy dataset at your disposal. In practice however, it is very rare to obtain the dataset you're going to analyze without cleaning values, creating new columns, reshaping the dataset, or transforming some variables. It is therefore essential to have some tools to perform these tedious tasks. 
> 
> Here comes `datawizard`, a lightweight package to easily manipulate, clean, transform, and prepare your data for analysis. It is part of the [easystats ecosystem](https://easystats.github.io/blog/), a suite of R packages to deal with your entire statistical analysis, from cleaning the data to reporting the results.

Reviewer 2: ```suggestion
`datawizard` is a lightweight package to easily manipulate, clean, transform, and prepare your data for analysis. It is part of the [easystats ecosystem](https://easystats.github.io/blog/), a suite of R packages to deal with your entire statistical analysis, from cleaning the data to reporting the results.

Most courses and tutorials about statistical modeling focus assume that you are working with a clean and tidy dataset. In practice, however, a major part of doing statistical modeling is preparing your data--cleaning up values, creating new columns, reshaping the dataset, or transforming some variables. `datawizard` provides easy to use tools to perform these common, critical, and sometimes tedious data preparation tasks. 
```

Author: done
